frontend/src/components/main-nav.js

```diff
@@ -130,10 +135,15 @@ class MainNav extends React.Component {
       margin-bottom: 2rem;
     }
     
-    .icon-caption:hover {
+    .icon-caption:hover:not(.icon-caption-active){
```

Comment:
hover should be on top of the active because it's still clickable and they can do click if if they want. otherwise it looks broken (so we don't want the :not part)